app/controllers/sections_controller.rb

```diff
@@ -12,10 +12,9 @@ def show
     authorize @section
     if @section.name == 'Eventos & Pesquisa'
       show_events
-    elsif @section.name == 'Cursos Pangeas'
-      show_courses
     elsif @section.name == section_life
       show_section
+    else redirect_to courses_path
```

Comment:
Why do we need this redirect?

Reply:
At sections#index, we have a list of the 3 sections. Before, the third one was rendering `sections#show_course`, but I moved that page to `courses#index` and redirected it.